### ISSUE TITLE
angular-animate

### ISSUE THREAD

**Issue by TWKDreamState** [ndxbxrme/generator-angular-meteor#6]
Hi,

I have added angular-animate (from the official Atmosphere package) and it breaks the app. The search, pagination and ordering causes the view to keep adding more items to the end of the list! Any idea on why this might be the case?

If I remove it as a dependency from the app.ng.js file, it goes back to working normally.

Many thanks.

EDIT: Interestingly, if I include it with Bower instead of the Atmosphere package, it seems to work ok. Version conflict?

**Comment by ndxbxrme** [comment]
hey
I have successfully used angular-animate in a meteor app but by that time I've always already removed the boilerplate code.
I'll give it a test and check what is going wrong.
What configuration were you using (coffee/js etc)?

**Comment by TWKDreamState** [comment]
Many thanks for the prompt response.

I used js, HTML, CSS, pagination support, Bootstrap, Bower, user auth (no oAuth).

I added angular-animate with ..

meteor add angularjs:angular-animate

..and then added the 'ngAnimate' to the app.ng.js.

I'm fairly new to meteor.

**Comment by TWKDreamState** [comment]
I think it is adding a second version of Angular to the app which might be causing the issues - I'm not sure how to prevent this.

**Comment by ndxbxrme** [comment]
have you got the latest version of the generator (v0.10.0)?
there was a fix for the multiple angular issue

**Comment by TWKDreamState** [comment]
I downloaded it yesterday but I will check - was the fix just for Bower or for Atmosphere packages as well?

**Comment by ndxbxrme** [comment]
If you got it yesterday then it should be the latest.
I've just successfully added angular-animate to a coffee/jade/stylus project, I'm trying JS/HTML/CSS now.

**Comment by TWKDreamState** [comment]
Hi,

I've just updated the generator (I think it was already the latest version).

I created a new project from scratch, added in the angular-animate atmosphere package (not bower), added the ngAnimate dependency into the app.ng.js and got the same result :( Double-checked in IE and Chrome. Still a second version of angular is being added from the atmosphere packages and still the weird behavior with the pagination, sorting, etc. Is this a problem with the atmosphere package?

**Comment by ndxbxrme** [comment]
This works for me http://ndxbxrme-animatejstest.meteor.com/
https://github.com/ndxbxrme/issue-animatejs
Could you push your project to git?

**Comment by TWKDreamState** [comment]
Ok,

When I install angular-animate from Atmosphere, it downloads and installs angularjs:angular version 1.3.15, whereas urigo:angular installs angular:angular 1.4.2 (not angularjs:angular). It this possibly where the issue is being caused? Would it be better to use bower to install?

**Comment by ndxbxrme** [comment]
I used
meteor add angular:angular-animate
I don't think the angularjs ones are up to date

**Comment by TWKDreamState** [comment]
I think I see my error. Let me try again. I should be installing with meteor add angular:angular-animate not meteor add angularjs:angular-animate.

**Comment by TWKDreamState** [comment]
Ah, you beat me to it!!

Many thanks for helping out :)

**Comment by ndxbxrme** [comment]
no probs
